File: src/frontend/src/app/models/Info.ts
export class Info {
    private _id: number = 1;
    private _phone: string = "";
    private _whatsapp: string = "";
    private _email: string = "";
    private _address: string = "";

    constructor(id: number, phone: string, whatsapp: string, email: string, address: string) {
        this._id = id;
        this._phone = phone;
        this._whatsapp = whatsapp;
        this._email = email;
        this._address = address;
    }

    get id(): number {
        return this._id;
    }

    set id(value: number) {
        this._id = value;
    }

    get phone(): string {
        return this._phone;
    }

    set phone(value: string) {
        this._phone = value;
    }

    get whatsapp(): string {
        return this._whatsapp;
    }

    set whatsapp(value: string) {
        this._whatsapp = value;
    }

    get email(): string {
        return this._email;
    }

    set email(value: string) {
        this._email = value;
    }

    get address(): string {
        return this._address;
    }

    set address(value: string) {
        this._address = value;
    }
}

File: src/backend/java/org/example/Angular/entities/Info.java
package org.example.Angular.entities;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class Info {

    @Id
    private int id;

    private String email;
    private String phone;
    private String whatsapp;
    private String address;

    public Info(int id) {
        this.id = id;
    }

    public Info() {

    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getWhatsapp() {
        return whatsapp;
    }

    public void setWhatsapp(String whatsapp) {
        this.whatsapp = whatsapp;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }
}
